tests/fast-integration/Makefile

```diff
@@ -30,3 +30,13 @@ ci-post-upgrade:
 	npm install
 	npm run upgrade-test-tests
 	npm run upgrade-test-cleanup
+
+.PHONY: ci-application-connectivity-2
```

Comment:
why there is a "-2" everywhere?

Reply:
Because these are pipelines related to Application Connectivity 2.0 components